sample/itembox/itembox_sample.json

```diff
@@ -0,0 +1,17 @@
+// Target-Tables: ESS_MS_Controls_ItemBox, ESS_MS_Controls_ItemBox_Profiles
+{
+  "id": "<unique-id>",
+  "name": "<internal-name>",
+  "title": "<localized-name/title>",
+  "description": "<localized-description>",
+  "profiles": [
+    {
+      "displayName": "<localized-profile-name>",
+      "type": "grid",
+      "grid": "<grid-name>",
+      "isActive": true,
+      "defaultSearchString": "%",
+      "regex": null
+    }
+  ]
+}
```

Comment:
itembox has autoincrement as of now, replace with guid?

Reply:
Detect update case by "SelectStatement"